lib/paperclip.rb

```diff
@@ -97,7 +97,8 @@ def self.options
       :log_command => true,
       :swallow_stderr => true,
       :content_type_mappings => {},
-      :use_exif_orientation => true
+      :use_exif_orientation => true,
+      :read_timeout => 120
```

Comment:
> Actually, trailing commas are not preferred: https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas.

It's fine, as we are not making it worse in this PR, but paperclip subscribes to thoughtbot guidelines, encoded in Hound CI: https://github.com/thoughtbot/guides/blob/master/style/ruby/sample.rb#L46-L59.